KsApi/GraphSchema.swift

```diff
@@ -193,6 +193,12 @@ public enum Query {
     }
   }
 
+  public enum ProjectsConnection {
```

Comment:
What's this for? 👀

Reply:
Oh, this is a result of a rabbit hole I was in. 😅 Will remove this.